rclpy/rclpy/node.py

```diff
@@ -1644,6 +1644,15 @@ def get_node_names_and_namespaces(self) -> List[Tuple[str, str]]:
         with self.handle as capsule:
             return _rclpy.rclpy_get_node_names_and_namespaces(capsule)
 
+    def get_node_names_and_namespaces_with_security_contexts(self) -> List[Tuple[str, str, str]]:
+        """
+        Get a list of names, namespaces and security contexts for discovered nodes.
+
+        :return: List of tuples containing two strings: the node name and node namespace.
```

Comment:
```suggestion
        :return: List of tuples containing three strings: node name, node namespace and security context.
```

Reply:
Done in https://github.com/ros2/rclpy/pull/538/commits/98f2e51f1a8e6f0ec3b1971a2474046865bc68d9.